index.html

```diff
@@ -703,7 +748,7 @@ <h2 id="payment-full-dto">PaymentFullDto</h2>
                 </thead>
                 <tbody>
                     <tr>
-                        <td>id</td>
+                        <td>Id</td>
```

Comment:
while we're at it. we could also caps 'Id' on line 728

Reply:
updated